### DISCUSSION TITLE
Does onnxruntime-web support XNNPACK backend?

### DISCUSSION THREAD

**Discussion by vadimkantorov** [microsoft/onnxruntime#16724] [General]
There is no mention of it here in the docs:
https://github.com/microsoft/onnxruntime/tree/main/js/web#readme but https://github.com/microsoft/onnxruntime/issues/13912 suggests that it might be supported somehow (but maybe not tested?)

XNNPACK supports wasm-simd backend and quantization and is the main backend of tflite

**Comment by xadupre** [comment]
It should support XNNPack. This page may be out of date. You can track the release changes here: https://github.com/microsoft/onnxruntime/releases. Option `--use_xnnpach` triggers it.
